corvid/enums/roadmap.md

```
@@ -0,0 +1,25 @@
+# Enums roadmap
+
+Status and next steps for `corvid/enums`, following the registration-mechanism
+migration.
+
+### 1. cstring_view accessor (optional)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)